functions/imagemagick/src/main/java/com/example/functions/imagemagick/ImageMagick.java

```diff
@@ -74,7 +81,7 @@ public void accept(GcsEvent gcsEvent, Context context) {
       List<AnnotateImageResponse> responses = response.getResponsesList();
       for (AnnotateImageResponse res : responses) {
         if (res.hasError()) {
-          LOGGER.info(String.format("Error: %s\n", res.getError().getMessage()));
+          LOGGER.info(String.format("Error: %s%n", res.getError().getMessage()));
```

Comment:
we typically don't need ending newline for log messages.

Reply:
+1

Reply:
Done.